nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/impl/OperationServiceImpl.java

```diff
@@ -496,6 +512,27 @@ public boolean isTypeAdaptable(Class<?> typeToAdapt, Class<?> targetType) {
         return result;
     }
 
+    /**
+     * Returns a simple copy of an {@link OperationDocumentation} to be used in the export.
+     * <p>
+     * Array fields of {@code od} are shallow copied.
+     */
+    private OperationDocumentation computeAliasOperationDocumentation(String alias, OperationDocumentation od) {
```

Comment:
Move this code to a constructor of `OperationDocumentation` so that the caller can just do `new OperationDocumentation(documentation, alias)`